core/src/main/web/app/mainapp/mainapp.js

```diff
@@ -616,8 +641,18 @@
               disconnected = true;
             },
             onReconnected: function() {
-              bkSessionManager.reconnectEvaluators();
-              disconnected = false;
+              bkSessionManager.isSessionValid()
```

Comment:
Oh, I see.  So when a beaker instance crashes, the client is already successfully reconnecting to a new instance?  (it's just that session is not valid anymore)?